app/poros/events.rb

```diff
@@ -0,0 +1,31 @@
+class Events
```

Comment:
because an instance of this represents a single "event", this should be singular 👍